pkg/codegen/schema/schema.go

```diff
@@ -1631,8 +1631,10 @@ type ResourceSpec struct {
 	// PlainInputs was a list of the names of the resource's plain input properties. This property is ignored:
 	// instead, property types should be marked as plain where necessary.
 	PlainInputs []string `json:"plainInputs,omitempty" yaml:"plainInputs,omitempty"`
-	// StateInputs is an optional ObjectTypeSpec that describes additional inputs that mau be necessary to get an
+	// StateInputs is an optional ObjectTypeSpec that describes additional inputs that may be necessary to get an
 	// existing resource. If this is unset, only an ID is necessary.
+	//
+	// `.get` functions will be generated only if StateInputs is non-nil.
```

Comment:
I don't believe this is true? We generate `get` even if just properties is set, I just checked some of the conformance test SDKs and they have 'get' functions but I definitely didn't fill StateInputs in on any of them yet.

Reply:
Hmm. Maybe not. I remember checking that but I don't image it changed.

https://github.com/pulumi/pulumi/blob/1416a34e69e6939b7e2ff1415e247a8ac52342de/pkg/codegen/nodejs/gen.go#L641-L647

Reply:
Then this is just a typo fix.